.github/workflows/static-analysis.yml

```diff
@@ -3,7 +3,7 @@ name: CBPay-JS tests
 on:
   pull_request:
     branches:
-      - 'master'
+      - 'main'
```

Comment:
Residual from the `master -> main` rename.

Reply:
we could also just remove this and let it run on all PRs, not just ones to main

Reply:
✔️